src/perf/lib/RpsClient.cpp

```diff
@@ -145,7 +143,7 @@ RpsClient::Start(
                 Registration,
                 Handler,
                 this,
-                &Connection);
+                &Connections[i].Handle);
```

Comment:
If the HQUIC operator returned a HQUIC& then the .Handle wouldn't be necessary here, right?

Reply:
I would prefer not. Not a big fan of non const references.

Reply:
Ok. We can keep as is for now.